### ISSUE TITLE
Failed to load the BatchNorm Block in ResNet_DUC_HDC_CityScapes-symbol.json

### ISSUE THREAD

**Issue by Serian1992** [TuSimple/TuSimple-DUC#13]
Hi, I'm very interested in your work. But when I tried to repeat your experiment, a MXNetError appeared that Cannot find argument 'fix_linear_trans' when loading the ResNet_DUC_HDC_CityScapes-symbol.json.  My current environment is Python 2.7, Mxnet version 1.0.0 under Win10.  Is this error because of the operating system or the different Mxnet version ?

**Comment by GrassSunFlower** [comment]
It was due to MxNet version mismatch. The symbol file was uploaded like 1 year ago.
There are three ways of fixing this problem:
1. Try using MxNet from our fork: https://github.com/TuSimple/mxnet
2. Or you can re-download the updated symbol file later. I'll update the file by tomorrow.

**Comment by Serian1992** [comment]
Thanks a lot! I may need the updated symbol file. Also, to verify the efficiency of HDC and DUC, I would like to reinitialize and train the  ResNet_DUC_HDC_CityScapes right from the baseline model ResNet-101 just like you, but seems like that only the well-reinited model is provided. Could you please also provide the source code on initializing the baseline model? Much appreciate.

**Comment by GrassSunFlower** [comment]
The symbol is already updated, you can check the download link.
As for the pretrained model, please see #1
